pyproject.toml

```diff
@@ -75,17 +75,17 @@ include = [
 # from `tests` so the root directory must be added to Python's path for editable installations
 dev-mode-dirs = ["."]
 
-# Black section
-# Read more here: https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-via-a-file
-[tool.black]
+# Ruff section
+# Read more here: https://docs.astral.sh/ruff/linter/#rule-selection
+[tool.ruff]
 line-length=80
 
-# Isort section
-# Read more here: https://pycqa.github.io/isort/docs/configuration/config_files.html
-[tool.isort]
-profile="black"
-line_length=80
-known_first_party = ["tuf"]
+[tool.ruff.lint]
+ignore = ["D400","D415","D213","D205","D202","D107","D407","D413","D212","D104","D406","D105","D411","D401","D200","D203"]
```

Comment:
note that these currently do not get used at all since "D" (pydocstyle) is not in the default rule set.

Maybe it's fine to leave them here unused for now: Enabling pydocstyle probably requires figuring out how to limit that to tuf/ only: tests are likely to not pass this.

Reply:
per-file-ignores accepts wildcards so if you enable the "D" ruleset, you could then try something like 

```
[tool.ruff.lint.per-file-ignores]
"tests/*" = ["D"]
"examples/*/*" = ["D"]
```